_data/sidebars/marketplace_dev_sidebar.yml

```diff
@@ -429,6 +429,11 @@ entries:
             include_versions:
             - "202108.0"
 
+          - title: Merchant Portal - Marketplace Customer Specific Prices feature integration
```

Comment:
Please use the correct feature name according to https://release.spryker.com/features/view/136 here & elsewhere. (Please do not forget to replace the links)